packages/suite/src/components/suite/NavigationBar/components/DeviceSelector/index.tsx

```diff
@@ -24,15 +23,6 @@ const Wrapper = styled.div<{ triggerAnim?: boolean }>`
         border-radius: 4px;
         box-shadow: 0 1px 2px 0 ${props => props.theme.BOX_SHADOW_BLACK_20};
     }
-
```

Comment:
you should remove the leftovers as well. all occurrences of `triggerAnim` variable

Reply:
79cc5e321064344c200478e87eedb235a1cd180c

Reply:
I take that back, more removal is needed.